llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgTransform/CMakeLists.txt

```diff
@@ -16,20 +6,6 @@ function(_add_transform_dialect_extension)
   add_dependencies(mlir-headers IREELinalgTransformExtIncGen)
 endfunction()
 
-function(_add_transform_doc)
-  set(LLVM_TARGET_DEFINITIONS LinalgTransformOps.td)
-  mlir_tablegen(LinalgTransformOps.md -gen-dialect-doc)
-  set(GEN_DOC_FILE ${IREE_DIALECTS_BINARY_DIR}/docs/Dialects/LinalgTransformOps.md)
-  add_custom_command(
-          OUTPUT ${GEN_DOC_FILE}
-          COMMAND ${CMAKE_COMMAND} -E copy
-                  ${CMAKE_CURRENT_BINARY_DIR}/LinalgTransformOps.md
-                  ${GEN_DOC_FILE}
-          DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/LinalgTransformOps.md)
-  add_custom_target(LinalgTransformOpsDocGen DEPENDS ${GEN_DOC_FILE})
-  add_dependencies(iree-dialects-doc LinalgTransformOpsDocGen)
-endfunction()
```

Comment:
Can you also remove references to this dialect from the documentation generation script?

https://github.com/openxla/iree/blob/14ce232df01b8606a7f2c8820050008200128afc/docs/website/generate_extra_files.sh#L48-L62

Should just be a matter of removing the two lines that reference `LinalgTransformOps`.

Reply:
I missed those, thanks